sources/packages/forms/src/form-definitions/supportingusersparent2022-2023.json

```diff
@@ -3317,253 +3459,56 @@
-                      "persistent": true,
-                      "clearOnHide": true,
-                      "refreshOn": "",
-                      "redrawOn": "",
-                      "tableView": false,
-                      "dataGridLabel": false,
-                      "labelPosition": "top",
-                      "description": "",
-                      "errorLabel": "",
-                      "tooltip": "",
-                      "hideLabel": false,
-                      "tabindex": "",
-                      "disabled": false,
-                      "autofocus": false,
-                      "dbIndex": false,
-                      "customDefaultValue": "",
-                      "calculateValue": "",
-                      "calculateServer": false,
-                      "widget": null,
-                      "validateOn": "change",
-                      "validate": {
-                        "required": false,
-                        "custom": "",
-                        "customPrivate": false,
-                        "strictDateValidation": false,
-                        "multiple": false,
-                        "unique": false
-                      },
-                      "allowCalculateOverride": false,
-                      "encrypted": false,
-                      "showCharCount": false,
-                      "showWordCount": false,
-                      "allowMultipleMasks": false,
-                      "id": "ekpksf"
-                    },
-                    {
-                      "label": "Collection and use of information",
-                      "tag": "p",
-                      "className": "",
                       "attrs": [
                         {
                           "attr": "",
                           "value": ""
                         }
                       ],
-                      "content": "<strong>Collection and use of information</strong>",
-                      "refreshOnChange": false,
-                      "customClass": "",
-                      "hidden": false,
-                      "modalEdit": false,
-                      "key": "collectionAndUseOfInformation",
-                      "tags": [],
-                      "properties": {},
-                      "conditional": {
-                        "show": null,
-                        "when": null,
-                        "eq": "",
-                        "json": ""
-                      },
-                      "customConditional": "",
-                      "logic": [],
-                      "attributes": {},
-                      "overlay": {
-                        "style": "",
-                        "page": "",
-                        "left": "",
-                        "top": "",
-                        "width": "",
-                        "height": ""
-                      },
+                      "content": "<strong>StudentAid BC Declaration</strong>",
```

Comment:
This must be in primary color as per figma.

Reply:
ok. Fixed. Thanks!